src/Backups/RestoreCoordinationRemote.cpp

```diff
@@ -95,8 +95,14 @@ void RestoreCoordinationRemote::setStage(const String & new_stage, const String
     stage_sync->set(current_host, new_stage, message);
 }
 
+void RestoreCoordinationRemote::setStageForCluster(const String & new_stage)
+{
+    stage_sync->setStageForCluster(new_stage);
+}
+
 void RestoreCoordinationRemote::setError(const Exception & exception)
 {
+    stage_sync->setStageForCluster(Stage::ERROR);
```

Comment:
`stage_sync->setError()` already adds a ZooKeeper node `<backup_path>/stage/error` with information about the error. Why isn't it enough?

Reply:
In some of our CI fails, I see that zookeeper is not able to delete all the nodes. And at times, it deletes the error node, but the stage (which was added for concurrency check) remains, causing misinterpretation.